appEnv.js

```diff
@@ -33,4 +33,9 @@ export const APP_BASE = IS_LOCAL
         : process.env.NEXT_PUBLIC_VERCEL_BRANCH_URL
     }`
 
-export const NEXT_PUBLIC_SEGMENT_WRITE_KEY = process.env.NEXT_PUBLIC_SEGMENT_WRITE_KEY
\ No newline at end of file
+export const NEXT_PUBLIC_SEGMENT_WRITE_KEY =
+  process.env.NEXT_PUBLIC_SEGMENT_WRITE_KEY
+
+export const NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY =
+  process.env.NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY ||
+  pk_test_51P8p2Y1taBPnTqn4IacUdFzw2mWPe8ljraPrpMlqMxtb8h1EvYTJvdGrj3kSeRIqm2ltL8RE8bAZL3EsLqpW3VNS00VZLcvudS
```

Comment:
You want this hard-coded and in the VCS?

I dunno.. seems like something that should stay in secrets/env vars only to me 🤔

Reply:
@RavenHursT I looked up best practice, it's safe because it's the test key `pk_test_...` plus the publishable keys are used in the client-side requests so they're bound to be accessible. The corresponding live key `pk_live_...` is a necessary vercel env var so that it's not operating in test mode.

Reply:
I know it's just a test key.. I was referring more so to the possibility of unintentionally causing payments to go into the wrong env by mistake.. 

@tomer-tgp does this new payments approach leverage Stripe's "sandboxing" capabilities as well, or no?

Reply:
it does